.github/workflows/ci.yml

```
@@ -157,7 +157,7 @@ jobs:
             mkdir truffle_tests
             cd truffle_tests
             truffle unbox metacoin
-            manticore . --contract MetaCoin --workspace output
+            coverage -m manticore . --contract MetaCoin --workspace output
```

```suggestion
            coverage run -m manticore . --contract MetaCoin --workspace output
```
I always forget that `coverage` isn't a drop-in replacement for `python`, but `coverage run` usually works that way.